backend/tests/load_testing/scenarios/chat_load_test.py

```
@@ -0,0 +1,118 @@
+        # Set host from config if not provided via CLI
+        if not self.host:
+            config = get_config()
+            self.host = config.target_url
+
+
+# ============================================================
+# EVENT HANDLERS
+# ============================================================
+
+@events.test_start.add_listener
+def on_test_start(environment, **kwargs):
```

Ruff (0.14.10)

Unused function argument: `environment`

(ARG001)